### PR TITLE
fix(VApp): remove layout styles

### PR DESCRIPTION
**Pull request by websitevirtuoso**
## Description

Issue - https://github.com/vuetifyjs/vuetify/issues/18064

## Markup:

```vue
<template>
  <v-app>
    <v-container>
      <v-text-field v-model="msg" />
    </v-container>
  </v-app>
</template>

<script setup>
import { ref } from 'vue'

const msg = ref('Hello World!')
</script>
```

### CODE REVIEW

File: packages/vuetify/src/components/VApp/VApp.sass [hunk 1]
```diff
@@ -11,6 +11,5 @@
   flex-direction: column
   flex: 1 1 auto
   max-width: 100%
-  min-height: 100vh
```
Comment (KaelWD) [inline packages/vuetify/src/components/VApp/VApp.sass:14]: This is for browsers without dvh

Comment (websitevirtuoso) [reply packages/vuetify/src/components/VApp/VApp.sass]: Restore property

**Comment by KaelWD** [comment]
Screenshot_20230831_021619

**Comment by websitevirtuoso** [comment]
Ok I can implement and test example from johnleider. Will do soon

**Comment by websitevirtuoso** [comment]
I tried to debug a little more and extra information: we really don't need z-index even in parentLayout

template

```
<template>
  <v-app>
    <v-layout class="rounded rounded-md">
      <v-app-bar title="Application bar" />

      <v-navigation-drawer>
        <v-list>
          <v-list-item title="Navigation drawer"/>
        </v-list>
      </v-navigation-drawer>

      <v-main
        class="d-flex"
        style="min-height: 300px"
      >
        <v-layout class="rounded rounded-md">
          <v-app-bar title="Application bar" />

          <v-navigation-drawer>
            <v-list>
              <v-list-item title="Navigation drawer"/>
            </v-list>
          </v-navigation-drawer>

          <v-main
            class="d-flex align-center justify-center"
            style="min-height: 300px"
          >
            <v-text-field v-model="msg" />
          </v-main>
        </v-layout>
      </v-main>
    </v-layout>
  </v-app>
</template>

<script setup>
import { ref } from 'vue'

const msg = ref('Hello World!')
</script>
```

ex: component v-layout has own z-index and proper position relative
image

In my own application, I tried to find an option where the z-index works and applies but didn't find it in multiple different layouts
image
I think this old code from v2 and not relevant anymore. If somebody will have issue I am able to implement it back or with parentLayout condition

**Review by johnleider — APPROVED** [review]

**Pull request merged**